<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgtRegisterUser</class>
 <widget class="QWidget" name="WgtRegisterUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация пользователя EasyRent</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSurname">
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="eSurname"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="eName"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPatronymic">
     <property name="text">
      <string>Отчество</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ePatronymic">
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCountry">
     <property name="text">
      <string>Страна</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblCity">
     <property name="text">
      <string>Город</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbCountry"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbCity"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblUsername">
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblPassword">
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="eUsername">
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="ePassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblPasswordRepeat">
     <property name="text">
      <string>Повторите пароль</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="ePasswordRepeat">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="bRegister">
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
